windows/ReactTestApp/MainPage.cpp

```diff
@@ -82,13 +88,37 @@ namespace winrt::ReactTestApp::implementation
                 AppTitle().Text(to_hstring(manifest.value().displayName));
             }
 
+            auto keyboardAcceleratorKey = VirtualKey::Number1;
             for (auto &&c : components) {
                 MenuFlyoutItem newMenuItem;
+
                 newMenuItem.Text(winrt::to_hstring(c.displayName.value_or(c.appKey)));
                 newMenuItem.Click(
                     [this, component = std::move(c)](IInspectable const &, RoutedEventArgs) {
                         LoadReactComponent(component);
                     });
+
+                // Add keyboard accelerators for first nine (1-9) components
+                if (keyboardAcceleratorKey <= VirtualKey::Number9) {
+                    KeyboardAccelerator keyboardAccelerator;
+                    /*
+                    According to
+                    https://docs.microsoft.com/en-us/uwp/api/windows.system.virtualkeymodifiers?view=winrt-19041
+                    following should work, but it doesn't, so using casts for now:
+                    keyboardAccelerator.Modifiers(VirtualKeyModifiers::Control |
+                    VirtualKeyModifiers::Shift);
+                    */
+                    keyboardAccelerator.Modifiers(static_cast<VirtualKeyModifiers>(
+                        static_cast<uint32_t>(VirtualKeyModifiers::Control) |
+                        static_cast<uint32_t>(VirtualKeyModifiers::Shift)));
+
+                    keyboardAccelerator.Key(keyboardAcceleratorKey);
+
+                    newMenuItem.KeyboardAccelerators().Append(keyboardAccelerator);
+                    keyboardAcceleratorKey =
+                        static_cast<VirtualKey>(static_cast<uint32_t>(keyboardAcceleratorKey) + 1);
```

Comment:
`VirtualKey` is defined as `enum class VirtualKey : int32_t`, so technically, we should cast to `int32_t`.
```suggestion
                        static_cast<VirtualKey>(static_cast<int32_t>(keyboardAcceleratorKey) + 1);
```
The safer alternative is to use [`std::underlying_type_t`](https://en.cppreference.com/w/cpp/types/underlying_type), but I think this is fine as is.